Source/Examples/Buttons/Buttons.csproj

```diff
@@ -1,43 +1,43 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net5.0-Windows</TargetFramework>
-    <LangVersion>preview</LangVersion>
-    <OutputType>WinExe</OutputType>
-    <UseWindowsForms>true</UseWindowsForms>
-    <Configurations>Debug;Release;Nightly;Canary;Stable</Configurations>
-    <ApplicationIcon>Stable.ico</ApplicationIcon>
+	<TargetFramework>net5.0-Windows</TargetFramework>
```

Comment:
Why are these tab's Keep them as spaces to match in with the rest of the "style" used throughout Krypton code

Reply:
There might be something wrong with the tabbing config

Reply:
Can't work out why though?

![image](https://user-images.githubusercontent.com/949607/150640922-577acdf4-5766-48f1-9fbb-88c4821d27e6.png)

Reply:
Tabs as spaces can be set in Global IDE options, or via an `.editorConfig` in the heirarchy somewhere.
If this is an new install of the IDE and no config, then it will default to tabs.

Reply:
So, copy the `.editorConfig` file?